testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -392,8 +392,24 @@
     "SURFACE_PDB",  # 111 FCC lattice topology for NSGrid bug #2345
     "SURFACE_TRR",  # full precision coordinates for NSGrid bug #2345
     "DSSP",  # DSSP test suite
+    "WB_AD",
```

Comment:
Please add desriptors for all these files, the filenames themselves won't make a ton of sense a couple months down the line.